r2/r2/templates/printable.html

```diff
@@ -101,6 +101,9 @@
       rowclass += " deleted"
     if hasattr(what, "saved") and what.saved:
       rowclass += " saved"
+      if c.user.gold and hasattr(what, "saved_category"):
+        item_saved_category = "no-category" if not what.saved_category else what.saved_category
```

Comment:
I'm recommenting since I rebased for that green checkmark and to remove dupliace logical checks.

"no-category" seems better than just "saved-None" or "saved-null" or "saved-" or no rowclass added at all; and since categories can't have a - in them, there is no problem with confusing this with an actual category.